pkgs/tools/misc/pywb/default.nix

```
@@ -0,0 +1,207 @@
+{ lib, fetchFromGitHub, python39, stdenv }:
```

is 3.9 a hard requirement? 

```suggestion
{ lib, fetchFromGitHub, python3, stdenv }:
```

---

If you use the `callPackage` trick below:

```suggestion
{ lib, fetchFromGitHub, python, stdenv }:
```

---

It shouldn't be a hard requirement.